settings.yaml

```
@@ -657,6 +657,12 @@ spec:
           validation: "required|url"
           value: https://api.lixingyong.com/api/images
           help: 随机图 API 地址。
+        - $formkit: text
+          name: rimage_custom_param_name
+          label: 随机图 API 资源标识变量
+          validation: "required"
+          value: postid
+          help: 随机图 API 资源标识变量，默认为postid，可根据随机图 API 的需要自行设置。
```

建议仅在选择文章支持使用随机图后再展示此项。

```suggestion
        - $formkit: text
        	if: "$get(rimage_cover_open).value == true"
          name: rimage_custom_param_name
          label: 随机图 API 资源标识变量
          validation: "required"
          value: postid
          help: 随机图 API 资源标识变量，默认为 postid，可根据随机图 API 的需要自行设置。
```

---

位置也可能需要调整一下，建议放在 `rimage_cover_open` 即文章使用随机封面图下面，防止用户选择之后再其上突然出现一个输入框造成误解。